mParticle-ComScore/MPKitComScore.m

```diff
@@ -148,14 +161,14 @@ - (instancetype)initWithConfiguration:(NSDictionary *)configuration startImmedia
 }
 
 - (MPKitExecStatus *)beginSession {
-    [CSComScore onUxActive];
+    [SCORAnalytics notifyUxActive];
```

Comment:
I think android is doing the same thing. They have separate methods for communicating foreground and background transitions. These are supposed to related to presenting content to the user.